mathics/builtin/box/layout.py

```diff
@@ -50,8 +56,8 @@ def to_boxes(x, evaluation: Evaluation, options={}) -> BoxElement:
 class ButtonBox(BoxExpression):
     """
     <dl>
-    <dt>'ButtonBox[$boxes$]'
-        <dd> is a low-level box construct that represents a button in a
+      <dt>'ButtonBox[$boxes$]'
+      <dd> is a low-level box construct that represents a button in a
     notebook expression.
```

Comment:
I had previously messed up here and this line should be part of the line before, since our homegrown formatter doesn't handle reflowed lines.